src/libraries/Microsoft.PowerFx.Connectors/ODataVisitor.cs

```diff
@@ -0,0 +1,237 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.Globalization;
+using Microsoft.PowerFx.Core.IR;
+using Microsoft.PowerFx.Core.IR.Nodes;
+using Microsoft.PowerFx.Core.IR.Symbols;
+using Microsoft.PowerFx.Core.Texl.Builtins;
+using Microsoft.PowerFx.Interpreter;
+using Microsoft.PowerFx.Types;
+
+namespace Microsoft.PowerFx.Connectors
+{
+    internal class ODataVisitor : IRNodeVisitor<string, DelegationRunContext>
+    {
+        internal static readonly ODataVisitor I = new ODataVisitor();
+
+        public override string Visit(TextLiteralNode node, DelegationRunContext runContext)
+        {
+            return $"'{node.LiteralValue}'";
```

Comment:
>{node.LiteralValue} [](http://example.com/codeflow?start=22&length=19)

Is there other escaping that needs to happen here? 
Eg, what if the node has a ' in it?

Reply:
Should these *LiteralNode visitors share code with SerializeLiteralValue?  (especially around some of the formatting)

Reply:
good point, refactored to use the same methods...also added escaping of `'` with double single quotes `''` per spec